### PR TITLE
Pr 18 review fixes

### PR DESCRIPTION
**Pull request by yogeshgadge**
Fixes as per the review comments on #18

### CODE REVIEW

File: src/app/applications/applications-menu/applications-menu.component.html [hunk 1]
```diff
@@ -38,7 +38,7 @@
     <cbp-applications-search></cbp-applications-search>
 
 
-    <div class="mat-menu-item cbp-applications-refresh" (click)="reloadApplicationsData($event)">
+    <div class="mat-menu-item cbp-applications-refresh" role="button" (click)="reloadApplicationsData($event)">
```
Comment (jscharett) [inline src/app/applications/applications-menu/applications-menu.component.html]: Not sure this solves the issue.  Is the div focusable?  also, does it have the right keyboard actions?  https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_button_role

Comment (yogeshgadge) [reply src/app/applications/applications-menu/applications-menu.component.html]: It does not have keyboard. It should.

Comment (yogeshgadge) [reply src/app/applications/applications-menu/applications-menu.component.html]: Assigned  #21 to @jscharett

**Comment by yogeshgadge** [comment]
Declining/Closing this PR. It will be addressed in #21

**Pull request closed without merging**